### ISSUE TITLE
Nested Folder Structure and Storage of Images

### ISSUE THREAD

**Issue by JamesLarkinWhite** [pbek/QOwnNotes#2312]
Dear all, i recently came accross this tool and would like to use it. 

I am not absolutely sure about the concept tho.

I tend to have multiple different places (nested strcture) were i store my notes. They are on various different places within my dir structure. Everything is stored as markdown. But i need to store the Iamges etc... within a seperate images Folder within the Folder of a specific note.

I think Vnote does this as well. So a for QOwnnotes a folder like QO_Images  and a File QO.json that keeps track of changes etc could be created in every folder. 

This is a mpockup of a possible strucutre

```
Home
│
└───PATH/TO/WORK-Folder                                                   <- All work related stuff is inside here
│   │
│   └───Notes
│   │   └───DailyNotes                                                              <- Daily Todolists etc ...
│   │   └───OtherNotes                                                            <- Specific Stuff that does not bekong to a Project 
│   └───+relPATH/TO/ProjectXY                                            
│   │   │  │
│   │   │  └───Code                                      
│   │   │   │
│   │   │   └─── ...                                  
│   │   │   │
│   │   │   └───notes                                                                <- This folder contains all my notes i take that belong to this project including metadata /pictiures....
│
└───PATH/TO/EDU                                                                     <- All my "Education"/ general knowlege gathering goes in here... there are subfolders with books i link to in the notes for example...
│   │
│   └───Notes
│   │   └───KnowledgeCards                                                       <- Main folder for stuff i want to learn about
│   │    │        └───Topic1                                                            <- For example all about DevOps is in one folder
│   │    └───Papers                                                                       <- Notes on all peer reviewd Publication i have read
```

In short:
- I have special places where i store my daily todos.md 
- I have a very nested structure
- I would want images to be stored in a subfolder within the same folder as the notes they belong to.

The reasion i do this, is that notes are where they belong. For example if a specific Project is done, i can archive it and all notes related to this Project are also inside. Is there a way to do this in QOwnnotes? 

Thank you in advance....

I am also open to suggestions what would be a good tool for me.

**Comment by pbek** [comment]
Hi there! 😁 I'm not sure what the question is, but if the question is "Can I use theses images in all my folders?" the answer is "Yes, sure!".

Those images should be viewed normally in the preview.

But if you are using QOwnNotes to add new images (e.g. by pasting them) those images will be created in the `<NoteFolder>/media` directory.

Does that answer your question?

**Comment by JamesLarkinWhite** [comment]
Almost yes. Currently i use [vnote](https://github.com/vnotex/vnote) but my organization has some rather serious problems with allowing me to use it on my work machine.

I think if i would have found this app first i would have likely used it from the start.

I actually quite like the look of QOwnNotes, but i feel that some of the features in VNote are missing (for my workflow) of course.
In Vnote you have to start with an empty folder and you have to add sub-folder via the app itself. I think each main folder is than related to a workspace. 
But images tat a pasted to Vnote are always saved in an automatically created  folder within the folder of a particular app. 
The advantage of this is of course, that you can more easily share your notes for specific projects. 

Thank you for your answer !

**Comment by pbek** [comment]
If that that answers your question feel free to close the issue (I'm not sure if you have another question or how I can help you. 😅)

**Comment by JamesLarkinWhite** [comment]
I think i am still not sure if my question is answered. I am sorry.
- Is the <NoteFolder>/media directory in the current folder of a particular note or is it saved at a central place? 
- Can i have notes in arbitrary folder on my System/Onedrive and see them via the QOwnNotes File tree view and search al notes globally?

I am sorry for beeing bad at asking questions... maybe i will just install it on my private Mashine and check.

**Comment by pbek** [comment]
> Is the /media directory in the current folder of a particular note or is it saved at a central place?

It's in your note folder, the root folder of your note structure.

> Can i have notes in arbitrary folder on my System/Onedrive and see tall notes i take via the QOwnNotes File tree view and search al notes globally?

You can create as many note folders you like in QOwnNotes, they can point to anywhere you like. You can only search for notes in the current note folder.

**Comment by BacLuc** [comment]
You can use a nested folder structure in your notes directory:
you just need to enable it in the settings:
![image](https://user-images.githubusercontent.com/1506818/174083802-573ab3da-86c8-4a47-b5dc-b3787abf20b9.png)
